apps/admin-x-activitypub/src/components/feed/FeedItemStats.tsx

```diff
@@ -28,6 +29,12 @@ const FeedItemStats: React.FC<FeedItemStatsProps> = ({
     const unlikeMutation = useUnlikeMutationForUser('index');
     const repostMutation = useRepostMutationForUser('index');
     const derepostMutation = useDerepostMutationForUser('index');
+    const {
+        displayValue: repostCountDisplay,
+        currentValue: repostCountNumber,
```

Comment:
How about aliasing `repostCount` thats passed in to `initialRepostCount`, then aliasing `currentValue` here to `repostCount`?

Reply:
Updated in https://github.com/TryGhost/Ghost/pull/22129/commits/12b9675cf6eab4609e5dc550959f42c809aabeff